doc/source/getting_started/install.rst

```diff
@@ -405,6 +433,8 @@ s3fs                      2021.05.0          Amazon S3 access
 Clipboard
 ^^^^^^^^^
 
+Must be manually managed, depending on operating system.
```

Comment:
It looks like PyQt4/5 and qtpy could be installed with a `clipboard` label. 

I think I would be `xclip` and `xsel` out of the table and into a `:note:` rst section saying that they need to be installed separately on Linux

Reply:
> Optional extras should work on all systems or not be included at all

Yeah agreed here.

Mainly for clarity, I was suggesting that `xsel` and `xclip` are separate to a paragraph noting these are not Python packages but OS packages one needs to install for Linux since each table above lists Python packages.

Reply:
Oh, gotcha.  100% agree on that.  Will change accordingly.